### ISSUE TITLE
Miitrix missing from the website

### ISSUE THREAD

**Issue by Sorunome** [matrix-org/matrix.org#1964]
Since the client section has been re-designed miitrix seems to be missing from the clients page

**Comment by awtj8o81ryywg793** [comment]
You're right; it seems that the client was lost. Sorry about that! I've opened [#3053](https://github.com/matrix-org/matrix.org/pull/3052) to fix this issue.
